backend/routes/admin/admin_db.py

```
@@ -411,3 +416,72 @@ def create_api_key(session: Session, team_id: int) -> Dict:
+    Get all demo users with their last played league, last submission timestamp,
+    and total number of submissions.
+
+    Returns:
+        List of dictionaries containing:
+        - team_name: Demo user's team name
+        - league_name: Last league the demo user played in
+        - last_submission: Timestamp of the last submission
+        - submission_count: Total number of submissions by the demo user
+    """
+    try:
+        demo_teams = session.exec(select(Team).where(Team.is_demo == True)).all()
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if Team.is_demo:` for truth checks

Replace with `Team.is_demo`

(E712)